tests/api/test_parameters.py

```diff
@@ -172,7 +172,7 @@ def test_path_parameter_someint__bad(simple_app):
     # non-integer values will not match Flask route
     app_client = simple_app.app.test_client()
     resp = app_client.get("/v1.0/test-int-path/foo")  # type: flask.Response
-    assert resp.status_code == 404
+    assert resp.status_code == 400, resp.text
```

Comment:
I believe 400 is more correct than 404 when a wrong type of path parameter is passed.